commit 12ea0dec4f06adf5738c1dd037666443799bbad9
Author: Bernhard M. Wiedemann <bwiedemann@suse.de>
Date:   Mon Jul 6 04:59:45 2020 +0200

    Use git commit date instead of build date
    
    and for distribution tarball builds, NEWS date is used as fallback.
    
    See https://reproducible-builds.org/ for why this is good.
    
    The first date call works with GNU date and FreeBSD date
    but has an unreproducible fallback.
    
    This PR was done while working on reproducible builds for openSUSE.

Index: paperjam-1.2.2/Makefile
===================================================================
--- paperjam-1.2.2.orig/Makefile
+++ paperjam-1.2.2/Makefile
@@ -1,6 +1,6 @@
 VERSION=1.2.2
 YEAR=2025
-BUILD_DATE:=$(shell date '+%Y-%m-%d')
+BUILD_DATE:=$(shell if [ -e .git ] ; then git log -1 --format=%cs ; else date -u -r NEWS '+%Y-%m-%d' 2>/dev/null || date '+%Y-%m-%d' ; fi)
 BUILD_COMMIT:=$(shell if git rev-parse >/dev/null 2>/dev/null ; then git describe --always --tags ; else echo '<unknown>' ; fi)
 
 DESTDIR=
